src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -71,11 +72,7 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   const { user, profile } = useUser();
   const partygoers = usePartygoers();
   const isVenueUsersLoaded = useIsVenueUsersLoaded();
-  const { experience } = useSelector((state) => ({
-    experience:
-      state.firestore.data.experiences &&
-      state.firestore.data.experiences[venueName],
-  }));
+  const experience = useSelector(experiencesSelector);
```

Comment:
`experiencesSelector` doesn't seem to have `venueName` selection logic. Is this change equivalent? Or is the `venueName` part outdated/broken code?

```
export const experiencesSelector = (state: RootState) =>
  state.firestore.data.experiences;
```

Reply:
https://github.com/sparkletown/sparkle/pull/1094/files#r562731404